tools/autograd/load_derivatives.py

```diff
@@ -43,32 +43,15 @@ def load_derivatives(derivatives_yaml_path: str, native_yaml_path: str) -> Seque
             assert str(function.func) not in functions_by_schema
             functions_by_schema[str(function.func)] = function
 
+        # Keep track of how many of which ops we've seen so we can
+        # disambiguate them with a numeric suffix.
+        op_prefixes = Counter[str]()
```

Comment:
Shouldn't this be `op_postfixes` ?

Reply:
It's kinda both. The keys are the prefixes and the values are the suffixes. Thoughts for a better name?

Reply:
prefix_to_suffix_map ?

Reply:
how about `op_counter`? i think that's even clearer.